Helper/Cart.php

```diff
@@ -1527,6 +1527,17 @@ private function getProductToGetImageForGroupedItem ($item)
         return $childProduct;
     }
 
+    /**
+     * Return user group id. "0" for guest users
```

Comment:
Is `0` as guest our creation or a default for Magento?

Reply:
Like our creation. Magento returns "1" for guest users and it's weird.

Reply:
Maybe it is worth explaining this in a comment then. There might be weird behaviour down the road one day and knowing this is our creation and not Magento's convention in a comment could be helpful.

Reply:
In fact, is there a reason we don't use `1`? We know it won't collide with another 1 since Magento reserves it for guest

Reply:
No, Magento uses "1" for guest users although "1" is real group_id. So in some cases we need to check if user logged in or not to know what this "1" mean. At least what I read in StackExchange today.
I will double-check this and update the comment.

Reply:
In Magento we use something like
```
 if ($customerSession->isLoggedIn()) {
      echo 'Customer Group Id: ' .  $customerSession->getCustomer()->getGroupId();
  }
```
So group_id started from 1 and make sense for logged in users only.

Reply:
Fixed comment. Could you please check @matt-thomason